instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -385,16 +384,6 @@ abstract class AbstractAws2ClientTest extends InstrumentationSpecification {
             <ResponseMetadata><RequestId>7a62c49f-347e-4fc4-9331-6e8e7a96aa73</RequestId></ResponseMetadata>
         </CreateQueueResponse>
         """
-    "Sqs"     | "SendMessage"       | "POST" | ""                    | "27daac76-34dd-47df-bd01-1f6e873584a0" | SqsClient.builder()     | { c -> c.sendMessage(SendMessageRequest.builder().queueUrl("someurl").messageBody("").build()) } | """
-        <SendMessageResponse>
-            <SendMessageResult>
-                <MD5OfMessageBody>d41d8cd98f00b204e9800998ecf8427e</MD5OfMessageBody>
-                <MD5OfMessageAttributes>3ae8f24a165a8cedc005670c81a27295</MD5OfMessageAttributes>
```

Comment:
I had to drop this test because of this hardcoded MD5. Alternative would be to run the test suite without the experimental message propgator enabled.

Reply:
If it helps you then setting a flag for only some tests is also quite easy
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4a0d7252afae67706426876fa807b0791b518e9c/instrumentation/netty/netty-4.1/javaagent/build.gradle.kts#L45-L68

Reply:
I think it is fine, as there are not the only tests that exercise sqs instrumentation. Idk whether this is what you are looking for, but to conditionally disable tests we usually use https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/ca24c388463cb36baa4b3926b26d7fc4b348b237/instrumentation/jaxrs/jaxrs-common/testing/src/main/groovy/AbstractJaxRsFilterTest.groovy#L45 For java tests it is pretty much the same but use junit5 `Assumptions` class.